src/main/java/org/jmxtrans/embedded/output/CopperEggWriter.java

```diff
@@ -434,49 +434,49 @@ public void write(Iterable<QueryResult> results) {
                 logger.warn("parts return NULL!!!");
             }
         }
-        if(jvm_os_counters.size() > 0) {
+        if(!jvm_os_counters.isEmpty()) {
```

Comment:
It looks to me like there is also a lot of redundancy that should be removed here. It is probably worth doing it at the same time as this refactoring.